src/UnitySampleProject/Assets/Scripts/Web3Accessor.cs

```diff
@@ -0,0 +1,38 @@
+using ChainSafe.GamingWeb3;
+using UnityEngine;
+
+public class Web3Accessor : MonoBehaviour
+{
+    static Web3Accessor instance;
+
+    Web3 web3;
+
+    public static Web3Accessor Instance
+    {
+        get
+        {
+            if (!instance)
+            {
+                var go = new GameObject("Web3Accessor");
+                DontDestroyOnLoad(go);
+                instance = go.AddComponent<Web3Accessor>();
+            }
+
+            return instance;
+        }
+    }
+
+    public Web3 Web3 => web3 ?? throw new System.Exception("Web3 instance not initialized");
```

Comment:
We could replace this with a static shortcut so that `Web3Accessor.Instance.Web3.Signer` would transform into a shorter version `Web3Accessor.Web3.Signer`.
```
public static Web3 Web3 => AssertAccessorNotNull().AssertWeb3NotNull();
// private static AssertAccessorNotNull returns Web3Accessor instance
// private AssertWeb3NotNull returns Web3 instance
```

Reply:
Why didn't I think of that? :|

Reply:
Still, no need for too methods to do that, just use a null-coalescing `?.` operator.